.editorconfig

```diff
@@ -15,7 +15,6 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.{json,yaml,yml,feature}]
```

Comment:
What's `feature`?

And JSON spaces 🤮 WordPress switched JSON files to tabs years ago 😏

https://github.com/WordPress/wordpress-develop/blob/trunk/.editorconfig

Reply:
> What's feature?

No idea. It must be coming from copypasta I did years ago :grin:

Reply:
Isn't `something.feature` a Behat/Gherkin test file?